systemd/v2.go

```diff
@@ -180,13 +180,8 @@ func unifiedResToSystemdProps(cm *dbusConnManager, res map[string]string) (props
 				newProp("TasksMax", num))
 
 		case "memory.oom.group":
-			// Setting this to 1 is roughly equivalent to OOMPolicy=kill
-			// (as per systemd.service(5) and
-			// https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html),
-			// but it's not clear what to do if it is unset or set
-			// to 0 in runc update, as there are two other possible
-			// values for OOMPolicy (continue/stop).
-			fallthrough
+			// This was set before the unit started, so no need to
```

Comment:
Dumb question, do we know this to actually be true? 

i.e. we've made it so we also process `memory.oom.group` in `Apply()`, which iiuc is getting its config for this from when you setup a new manager, but for `Set()` we're sort of hoping the user already set it up initially and just avoid warning on it. The downside here then is if someone doesn't have it configured for `Apply()`, and then adds it later with `Set()`, systemd won't be in the loop and a daemon-reload will cause systemd to overwrite with whatever its `OOMPolicy` is set to be for the unit?

I guess I don't have a better answer for these sort of "must be done on unit creation" type settings unless we can warn iff we know the systemd prop isn't aligned already.